pixi.toml

```diff
@@ -151,6 +151,9 @@ cpp-prepare-msvc = "cmake -G 'Visual Studio 17 2022' -B build-msvc -S ."
 
 
 [dependencies]
+# IMPORTANT: do not add any dependencies here that may break CI. All dependencies should be available on all supported
+# platforms (including linux-aarch64), or added conditionally.
```

Comment:
How can I check if a dependency is available on all supported platforms (i.e. won't break CI)?